src/constants/HIDmap.ts

```diff
@@ -5,6 +5,8 @@ export interface HidInfo {
   category: HIDCategory
   displayString: string
   webKeyId: string
+  whichID: number
+  locationID: number
```

Comment:
Please make this `locationID?: number` and only specify it on entries where it matters

Reply:
If it's undefined, you can just assume that it's 0, like it is for 99% of keys

Reply:
Fixed in: [643c014](https://github.com/WootingKb/wooting-macros/pull/194/commits/643c014ba8fdf17075af61d885d101f68b79c2c8)